PyInstaller/building/osx.py

```diff
@@ -189,14 +193,30 @@ def assemble(self):
         #
         # Move all directories from ./MacOS/ to ./Resources and create symlinks
         # in ./MacOS.
-        bin_dir =os.path.join(self.name, 'Contents', 'MacOS')
-        res_dir =os.path.join(self.name, 'Contents', 'Resources')
-        # Qt plugin directories does not contain data files.
-        ignore_dirs = set(['qt4_plugins', 'qt5_plugins'])
```

Comment:
The explicit checks for qt plugin dirs is not necessary anymore? Is it replaced by explicitly symlinking only data files?